docs/getting_started/DEBUGGING.md

```diff
@@ -0,0 +1,122 @@
+# Debugging
+
+## Fetching service logs
+
+### Option A: `bolt logs`
+
+To read logs of a **single, currently running service**, run the following command:
+
+```
+bolt logs my-service
+```
+
+This command also supports `--follow` and `--stderr`.
+
+### Option B: Loki
+
+If the container has **stopped, restarted, or is a short-lived job**, you need to use Loki to read the logs.
+
+Connect to the Grafana dashboard by running the following, then visiting http://localhost:9090:
+
+```
+./scripts/forward/grafana.sh
+```
+
+Open the _Explore_ page and query `{service=~"rivet-.*"} |= "ERROR"`, or click [here](http://localhost:9090/explore?panes=%7B%22DNM%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bservice%3D~%5C%22rivet-.*%5C%22%7D%20%7C%3D%20%5C%22ERROR%5C%22%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22code%22%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1).
```

Comment:
This might be a bit easier to follow:

````suggestion
Connect to the Grafana dashboard by running the following:

```
./scripts/forward/grafana.sh
```

Next, visit http://localhost:9090 and open the _Explore_ page and query `{service=~"rivet-.*"} |= "ERROR"`, or click [here](http://localhost:9090/explore?panes=%7B%22DNM%22:%7B%22datasource%22:%22loki%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bservice%3D~%5C%22rivet-.*%5C%22%7D%20%7C%3D%20%5C%22ERROR%5C%22%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22loki%22%7D,%22editorMode%22:%22code%22%7D%5D,%22range%22:%7B%22from%22:%22now-6h%22,%22to%22:%22now%22%7D%7D%7D&schemaVersion=1&orgId=1).
````